src/fides/api/ops/api/v1/endpoints/privacy_notice_endpoints.py

```diff
@@ -253,7 +262,9 @@ def prepare_privacy_notice_patches(
                 detail=f"No PrivacyNotice found for id {update_data.id}.",
             )
 
-        updates_and_existing.append((update_data, existing_notices[update_data.id]))
+        updates_and_existing.append(
+            (update_data.escaped(), existing_notices[update_data.id])
```

Comment:
this isn't properly typed—I put the `escaped()` method on the base method so it returns the base type, not the extended type, not sure how to work around that